pbjam/peakbag.py

```diff
@@ -374,59 +372,4 @@ def __call__(self, model_type='simple', tune=1500, nthreads=1, maxiter=4,
         except:
             self.summary = pm.stats.summary(self.samples)
             
-        self.par_names = self.summary.index
-
-
-#    def simple(self):
-#        """
-#        Creates a simple peakbagging model in PyMC3's self.pm_model which is
-#        an instance of pm.Model().
-#
-#        The simple model has three parameters per mode (freq, w, h) and one
-#        back parameter per rung of the frequency ladder.
-#
-#        All parameters are independent.  For a model with additional constraints
-#        see model_gp.
-#
-#        Priors on parameters are defined as follows:
-#            lX ~ Normal(log(start[lX]), dnu*0.1)
-#            widthX ~ Lognormal(log(start[widthX]), 1.0)
-#            heightX ~ Lognormal(log(start[height0]), 0.4)
-#            back ~ Lognormal(log(1), 0.5)
-#
-#        The equivalent likelihood function of the observed data is dealt with
-#        using a Gamma distribution where alpha=1 and beta=1/limit where limit
-#        is the model of the spectrum proposed.  Using this gamma distirbution
-#        is the equivalent of stating that observed/model is distributed as
-#        chi squared two degrees of freedom (see Anderson 1990 for more details).
-#        The use of the Gamma distribution is much more in line with the ethos of
-#        probabistic programming languages.
-#        """
-#
-#        dnu = 10**self.asy_result.summary.loc['dnu', 'mean']
-#        dnu_fac = 0.03 # Prior on mode frequency has width 3% of Dnu.
-#        width_fac = 1.0 # Lognorrmal prior on width has std=1.0.
-#        height_fac = 0.4 # Lognorrmal prior on height has std=0.4.
-#        back_fac = 0.4 # Lognorrmal prior on back has std=0.4.
-#        N = len(self.start['l2'])
-#
-#        with self.pm_model:
-#
-#            width0 = pm.Lognormal('width0', mu=np.log(self.start['width0']),
-#                                  sigma=width_fac, shape=N)
-#            width2 = pm.Lognormal('width2', mu=np.log(self.start['width2']),
-#                                  sigma=width_fac, shape=N)
-#
-#            l0 = pm.Normal('l0', self.start['l0'], dnu*dnu_fac, shape=N)
-#
-#            l2 = pm.Normal('l2', self.start['l2'], dnu*dnu_fac, shape=N)
-#
-#            height0 = pm.Lognormal('height0', mu=np.log(self.start['height0']),
-#                                    sigma=height_fac, shape=N)
-#            height2 = pm.Lognormal('height2', mu=np.log(self.start['height2']),
-#                                    sigma=height_fac, shape=N)
-#            back = pm.Lognormal('back', mu=np.log(1.0), sigma=back_fac, shape=N)
-#
-#            limit = self.model(l0, l2, width0, width2, height0, height2, back)
-#
-#            pm.Gamma('yobs', alpha=1, beta=1.0/limit, observed=self.ladder_s)
+        self.par_names = self.summary.index
```

Comment:
Missing a new line character

Reply:
Since when are you PEP8 compliant?

Do things break if we don't have an empty line at the end?